### PR TITLE
VEC-323 asd avs on gke tls rbac

### PR DESCRIPTION
**Pull request by arrowplum**
This will set up a TLS-enabled cluster with the admin:admin

Currently since the helm chart is not published there is a command line argument to point to a helm chart directory.

`./full-create-and-install.sh --chart-location /home/joem/src/helm-charts/aerospike-vector-search/`

### CODE REVIEW

File: basic-search/check.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import argparse
```
Comment (dwelch-spike) [inline basic-search/check.py:1]: Should this file be committed? It seems unnecessary.

**Review by davi17g — APPROVED** [review]
LGTM

File: kubernetes/full-create-and-install.sh [hunk 2]
```diff
@@ -18,166 +26,431 @@ print_env() {
...
+set_env_variables() {
+    export WORKSPACE="$(pwd)"
+    export PROJECT_ID="$(gcloud config get-value project)"
+    export CLUSTER_NAME="${PROJECT_ID}-avs-noauth-010"
+    export NODE_POOL_NAME_AEROSPIKE="aerospike-pool"
+    export NODE_POOL_NAME_AVS="avs-pool"
+    export ZONE="us-central1-c"
+    export FEATURES_CONF="$WORKSPACE/features.conf"
+    export BUILD_DIR="$WORKSPACE/generated"
+    export RUN_INSECURE=1
```
Comment (hev) [inline kubernetes/full-create-and-install.sh:43]: This flag name seems opposite of what I'd expect. I think it should be true if you are opting to turn off security, but currently false turns it off security. I like the idea of doing a weird negative opt in here this just doesn't seem right.

**Review by hev — APPROVED** [review]

File: kubernetes/full-create-and-install.sh (hunk 2 above)
Comment (hev) [reply kubernetes/full-create-and-install.sh:43]: I am actually not sure which does which, as I get the cert and password prompt for both values of the flag.

**Comment by arrowplum** [comment]
replaced with #60

**Pull request closed without merging**